frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1341,13 +1346,16 @@ function BookMapWidget:showMenu()
     }
     UIManager:show(button_dialog)
 end
+
+function BookMapWidget:onShowMenu()
+    self:showMenu()
+end
+
 function BookMapWidget:showAbout()
     local text = _([[
-Book map displays an overview of the book content.
+The book map provides a summary of the book's content, showing chapters and pages visually. If statistics are enabled, black bars represent pages already read (gray for pages read in the current session), with varying heights based on reading time. 
```

Comment:
I somehow read "Book map" as a thing, like "iPad Air", so "The book map" sounds odd to me.
Also, "provides a summary" feels misleading, summary implying a bunch of words to me.
"overview" (as a view over, from above, like a plan) feels more true to the feature.

"showing chapters and pages visually" summarize with less info the 2 list items you removed below. I get someone will understand that anyway, but still, again, style vs. precision.

Reply:
Is not always the case that the sum of its parts equals the meaning of a word, regarding overview: yes it originally meant that (view from above) but now that is archaic, on its way out, and the word means primarily “ to review, summarise, take an overview of.”

summary is “the essence or essential part of something”

Reply:
@Frenzie @NiLuJe : any more thoughts about these wording (this comment, and the one above, which has become `Page-bar width` since).

Reply:
Seems fine to me, provided it's all technically accurate but you'd have said something about that.

Reply:
You're ok with?
```diff
-Book map displays an overview of the book content
+The book map provides a summary of the book's content
```
You don't share my non-native-english feelings?

Reply:
You mean "Book map" as an entity? I would've said something about the lack of "the" if I felt it didn't work. I wouldn't change it myself.

Reply:
> I wouldn't change it myself.

Define `it`. The current wording, or the diff chunk in question in this P>R? (i.e., ack or nack on the change ;)).

Reply:
[for reference](https://learnenglish.britishcouncil.org/grammar/english-grammar-reference/definite-article), for a second forget about [this book map], and think of a map in your hands., which of these is correct?:
1. map says to turn left here
2. the map says to turn left here

interesting fact (VERY INFORMAL), calling something "sh*t" is to say something is really bad, but if you call something "The sh*t" is the opposite, something really, really good, the best. e.g the kindle oasis is "the sh*t".

Reply:
> I somehow read "Book map" as a thing, like "iPad Air",

Which of these is correct?:
Google Map says to turn left here
The Google Map says to turn left here

Reply:
"Google Maps" falls under the "We do not normally use the definite article with names" exception.

Reply:
So was my idea with "Book map".
There's no existing notion of a "map" with a book (unlike your turn left example).
There is something in KOReader that we called "Book map" and shows some "overview" of the book. It's not something anybody would intuitively call "the book map". It's just something that we the authors named "Book map".
(Of course, posterity will call it "the book map" in 20 years when all eBook readers will have to have that.)

Reply:
> (Of course, posterity will call it "the book map" in 20 years when all eBook readers will have to have that.)

humble yourself silly boy ;)

a map is simply a diagram, the term "book map" is no different then "street map" for example, although there might not be other implementations elsewhere (in other reading apps), the idea of a book map is not in itself novel, just a diagram of the content of said book.

Reply:
besides, calling something "The Something" is a way increasing its street cred (in a manner of speaking).

Reply:
> Define `it`. The current wording, or the diff chunk in question in this PR? (i.e., ack or nack on the change ;)).

I wouldn't change "book map" myself because it's a thing we decided on a couple of years ago that I'm fine with. It's like how in Excel you'd say "AutoSum does this and that" rather than "The AutoSum." But "the book map" sounds fine too.

Reply:
I don't buy these the shoes street cred/crap subtleties. That's now how we do marketing here !

Anyway, the only reason for going with "The book map" is that translators already went that way :/
"De paginakaart toont een overzicht van de boekinhoud.\n"
"O mapa do livro exibe uma visÃ£o geral do conteÃºdo do livro.\n"
"Die Buchkarte zeigt eine Ã~\bersicht Ã¼ber den Inhalt des Buches.\n" (at least, German kept the Uppercase :))
"El mapa del libro muestra una visiÃ³n general del contenido del libro.\n"
"La carte du livre affiche un aperÃ§u du contenu du livre.\n"
"Il navigatore delle pagine mostra miniature delle pagine.\n" (Italian mixed Book map with Page browser)

Reply:
so what is the final verdict here then...?

Reply:
do I change it or what?

Reply:
Let's go with "Book map" without the determiner. @poire-z wrote it after all. ;-)